### PR TITLE
Fix GHA test action to just run tests on Plone 5.2. Plone 6 CI is bes…

### PR DESCRIPTION
**Pull request by ale-rt**
…t done in Jenkins. Update to the tested Plone version to 5.2.6

Refs. #1255

### CODE REVIEW

**Comment by ale-rt** [comment]
@tisto I hope this PR makes my point more clear than what I wrote in https://github.com/plone/plone.restapi/pull/1255#issuecomment-955715001

Disclaimer I might have not have a full vision of the big picture but I am pretty convinced I nailed it.
If I am wrong, please be patient and explain!

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [3.8, 3.7, 3.6]
-        plone-version: ["6.0", "5.2"]
```
Comment (tisto) [inline .github/workflows/tests.yml:10]: @ale-rt as outlined in my comment I'd prefer to keep things as they are in tests.yml. The rest of the PR is fine. :)

**Review by tisto — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (ale-rt) [reply .github/workflows/tests.yml:10]: I know what you outlined, but I do not understand it :)
jenkins tests are already doing everything is needed and testing Plone 6.0 here is just adding complications.
In addition the tested Plone 6 version is already old, see:
- https://github.com/plone/plone.restapi/blob/52b2bcfbffa4559216843c37a19d088fd0e722f8/plone-6.0.x.cfg#L4

I am still convinced this PR is doing the right thing!

**Comment by ale-rt** [comment]
See the discussion that happened in #1255

**Pull request closed without merging**